diff -p -up CoinAll-1.3.1/ThirdParty/ASL/solvers/arithchk.c.orig CoinAll-1.3.1/ThirdParty/ASL/solvers/arithchk.c
--- CoinAll-1.3.1/ThirdParty/ASL/solvers/arithchk.c.orig	2010-01-21 20:09:25.680740133 -0200
+++ CoinAll-1.3.1/ThirdParty/ASL/solvers/arithchk.c	2010-01-21 20:08:51.879290006 -0200
@@ -122,10 +122,10 @@ ccheck(void)
 	long Cray1;
 
 	/* Cray1 = 4617762693716115456 -- without overflow on non-Crays */
-	Cray1 = printf(emptyfmt) < 0 ? 0 : 4617762;
-	if (printf(emptyfmt, Cray1) >= 0)
+	Cray1 = printf("%s", emptyfmt) < 0 ? 0 : 4617762;
+	if (printf("%s", emptyfmt, Cray1) >= 0)
 		Cray1 = 1000000*Cray1 + 693716;
-	if (printf(emptyfmt, Cray1) >= 0)
+	if (printf("%s", emptyfmt, Cray1) >= 0)
 		Cray1 = 1000000*Cray1 + 115456;
 	u.d = 1e13;
 	if (u.L == Cray1)
diff -p -up CoinAll-1.3.1/Clp/src/CbcOrClpParam.cpp.orig CoinAll-1.3.1/Clp/src/CbcOrClpParam.cpp
--- CoinAll-1.3.1/Clp/src/CbcOrClpParam.cpp.orig	2010-01-21 20:50:56.769293412 -0200
+++ CoinAll-1.3.1/Clp/src/CbcOrClpParam.cpp	2010-01-21 20:51:14.542290518 -0200
@@ -1075,7 +1075,7 @@ CoinReadNextField()
     }
 #else
     if (CbcOrClpReadCommand==stdin) {
-      fprintf(stdout,coin_prompt);
+      fprintf(stdout,"%s",coin_prompt);
       fflush(stdout);
     }
     where = fgets(line,1000,CbcOrClpReadCommand);
diff -p -up CoinAll-1.3.1/CoinUtils/src/CoinLpIO.cpp.orig CoinAll-1.3.1/CoinUtils/src/CoinLpIO.cpp
--- CoinAll-1.3.1/CoinUtils/src/CoinLpIO.cpp.orig	2010-07-12 20:21:38.806515374 -0300
+++ CoinAll-1.3.1/CoinUtils/src/CoinLpIO.cpp	2010-07-12 20:34:41.170513007 -0300
@@ -1294,7 +1294,8 @@ CoinLpIO::read_monom_row(FILE *fp, char
   char buff[1024], loc_name[1024], *start;
   int read_sense = -1;
 
-  sprintf(buff, start_str);
+  strncpy(buff, start_str, sizeof(buff));
+  buff[sizeof(buff) - 1] = 0;
   read_sense = is_sense(buff);
   if(read_sense > -1) {
     return(read_sense);
@@ -1393,7 +1394,8 @@ CoinLpIO::read_row(FILE *fp, char *buff,
   int read_sense = -1;
   char start_str[1024];
   
-  sprintf(start_str, buff);
+  strncpy(start_str, buff, sizeof(start_str));
+  start_str[sizeof(start_str) - 1] = 0;
 
   while(read_sense < 0) {
 
diff -p -up CoinAll-1.3.1/CoinUtils/src/CoinParamUtils.cpp.orig CoinAll-1.3.1/CoinUtils/src/CoinParamUtils.cpp
--- CoinAll-1.3.1/CoinUtils/src/CoinParamUtils.cpp.orig	2010-07-12 20:42:08.836516513 -0300
+++ CoinAll-1.3.1/CoinUtils/src/CoinParamUtils.cpp	2010-07-12 20:42:26.311511956 -0300
@@ -66,7 +66,7 @@ std::string nextField (const char *promp
     { where = fgets(line,1000,readSrc) ; }
 #else
     if (readSrc == stdin)
-    { fprintf(stdout,prompt) ;
+    { fprintf(stdout,"%s",prompt) ;
       fflush(stdout) ; }
     where = fgets(line,1000,readSrc) ;
 #endif
diff -p -up CoinAll-1.3.1/SYMPHONY/src/Master/main.c.orig CoinAll-1.3.1/SYMPHONY/src/Master/main.c
--- CoinAll-1.3.1/SYMPHONY/src/Master/main.c.orig	2010-07-12 21:01:54.853515428 -0300
+++ CoinAll-1.3.1/SYMPHONY/src/Master/main.c	2010-07-12 21:02:19.718512708 -0300
@@ -672,7 +672,7 @@ int sym_read_line(const char *prompt, ch
    
   while(true){
      strcpy(getl, "");
-     printf(prompt);
+     printf("%s", prompt);
      fflush(stdout);
      fgets(getl, MAX_LINE_LENGTH, stdin);
 
diff -p -up CoinAll-1.3.1/Ipopt/src/Common/IpJournalist.cpp.orig CoinAll-1.3.1/Ipopt/src/Common/IpJournalist.cpp
--- CoinAll-1.3.1/Ipopt/src/Common/IpJournalist.cpp.orig	2010-07-12 21:17:08.903516741 -0300
+++ CoinAll-1.3.1/Ipopt/src/Common/IpJournalist.cpp	2010-07-12 21:17:50.886610548 -0300
@@ -394,7 +394,7 @@ namespace Ipopt
   {
     DBG_START_METH("Journal::Print", 0);
     if (file_) {
-      fprintf(file_, str);
+      fprintf(file_, "%s", str);
       DBG_EXEC(0, fflush(file_));
     }
   }
